docs/docs/reference/metaprogramming/macros.md

```diff
@@ -192,8 +192,8 @@ quote but no splice between the parameter binding of `T` and its
 usage. But the code can be rewritten by adding a binding of a `Type[T]` tag:
 
 ```scala
-def to[T, R](f: Expr[T] => Expr[R])(using Type[T], Type[R], Quotes): Expr[T => R] =
-   '{ (x: T) => ${ f('x) } }
+def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T])(using Type[R], Quotes): Expr[T => R] =
+   '{ (x: $t) => ${ f('x) } }
```

Comment:
That is not valid syntax anymore. We probably need to update the test below.

Reply:
Tese two lines should still be reverted

Reply:
Oh, whoops, I didn't realize this hadn't been done -- thanks for following up

Reply:
Actually, in revisiting this -- What has changed in the syntax such that this approach is invalid? The previous paragraph talks about creating a binding for `Type[T]`, so simply reverting it isn't sufficient.

Just guessing,
```scala
def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T])(using Type[R], Quotes): Expr[T => R] =
   '{ (x: ${t.Underlying}) => ${ f('x) } }
```
seems like it would work, if interpolating a `t: Type[T]` directly is no longer correct

Reply:
Indeed, the start of that section should be

> ## Lifting Types
> Types are not directly affected by the phase consistency principle.
It is possible to use types defined at any level in any other level.
But, if a type is used in a subsequent stage it will need to be lifted to a `Type`.
Indeed, the definition of `to` above uses `T` in the next stage, there is a
quote but no splice between the parameter binding of `T` and its
usage. But the code can be rewritten by adding an explicitly binding of a `Type[T]`:
> ```scala
> def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T])(using Type[R], Quotes): Expr[T => R] =
>   '{ (x: t.Underlying) => ${ f('x) } }
> ```

Reply:
👍